ods_ci/tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -201,16 +201,16 @@ Verify Favorite Resource Cards
     ${list_of_tile_ids} =    Get List Of Ids Of Tiles
     Verify Star Icons Are Clickable    ${list_of_tile_ids}
 
-    ${favorite_ids} =    Get Slice From List    ${list_of_tile_ids}    ${27}    ${48}
+    ${favorite_ids} =    Get Slice From List    ${list_of_tile_ids}    ${2}    ${7}
     Add The Items In Favorites    @{favorite_ids}
 
     ${list_of_tile_ids} =    Get List Of Ids Of Tiles
-    Favorite Items Should Be Listed First    ${favorite_ids}    ${list_of_tile_ids}    ${21}
+    Favorite Items Should Be Listed First    ${favorite_ids}    ${list_of_tile_ids}    ${5}
```

Comment:
BTW do we know why we have this change? Is there some tracking issue for reference that can explain the intention?

Reply:
There are less numbers of cards available now. Not sure about a known change that explains the reason behind it.

Reply:
21 to 5 seems a big leap, though.

Reply:
Hmm, if this is the reason, it's a pitty it's not here already, since it's a 3 line change :thinking: :slightly_smiling_face:

Reply:
>number of Resources cards available, I changed the test to support it as well

does it mean the test will fail on RHOAI then?

Reply:
No, it does not fail on RHOAI, since in RHOAI there are more favorite cards, so `Get Slice From List` will work as excpected.
I will change the test logic in another PR, to avoid retesting it on ODH cluster again, and more rebases.